go-controller/pkg/cni/cnishim.go

```diff
@@ -209,11 +209,18 @@ func (p *Plugin) CmdAdd(args *skel.CmdArgs) error {
 			klog.Error(err.Error())
 			return err
 		}
+
+		podInterfaceInfo := response.PodIFInfo
+		if err := pr.checkOrUpdatePodUID(podInterfaceInfo.PodUID); err != nil {
```

Comment:
@cathy-zhou is this necessary now that cni.go fills in the pod UID for the unprivileged case?